src/utils.h

```diff
@@ -47,3 +49,5 @@ gboolean ayatana_common_utils_execute_command(const gchar * cmd);
 gboolean ayatana_common_utils_open_url(const gchar * url);
 gboolean ayatana_common_utils_have_program(const gchar * program);
 gboolean ayatana_common_utils_zenity_warning(const char *icon_name, const char *title, const char *text);
+
+void ayatana_common_utils_elipsize(char *sText);
```

Comment:
Urgh, don't split prototype definition and code for it into two different commits. Please unite them.